### PR TITLE
MQTT read_table_A.pl support

### PR DESCRIPTION
**Pull request by jsiddall**
This was written by Giles Godart-Brown

### CODE REVIEW

**Comment by hollie** [comment]
This pull request passes the Travis check, but the result is not reflected in the status above due to a change I had to do from travis-ci.org to travis-ci.com.

The actual result of the test is here: https://travis-ci.com/github/hollie/misterhouse/builds/209211100

So CI-wise, the pull request is OK to merge.

**Comment by hollie** [comment]
Hello @jsiddall,

thanks for this pull request. 

Maybe a small cosmetic suggestions: in MQTT 'speak' the server is called 'broker'. 

Would it make sense to keep the name of the instance in line with the commonly used term, especially in the table definition?

So instead of calling the instance `MQTT_GATEWAY` I think it makes sense to rename it to `MQTT_BROKER`.

What do you think?

Best regards,
 Lieven.

File: lib/read_table_A.pl [hunk 1]
```diff
@@ -1901,7 +1901,39 @@ sub read_table_A {
         $object = '';
     }
     #-------------- End AoGSmartHome Objects ----------------
-
+    #-------------- MQTT Objects -----------------
+    elsif ( $type eq "MQTT_GATEWAY" ) {
```
Comment (hollie) [inline lib/read_table_A.pl:1905]: I would replace `MQTT_GATEWAY` with `MQTT_BROKER` here as that is the terminology that is used in MQTT-speak.

**Comment by jsiddall** [comment]
I replaced all occurrences of gateway with broker and amended the commit.

**Comment by hollie** [comment]
Thank you Giles! Merging in...

**Pull request merged**